ffi/build.rs

```diff
@@ -41,10 +57,12 @@ fn main() {
                 }
             }
         }
-    } else {
-        fs::remove_file("mitmproxy_rs/windows-redirector.exe").ok();
-        for wd_file in windivert_files {
-            fs::remove_file(format!("mitmproxy_rs/{wd_file}")).ok();
-        }
+    }
+    #[cfg(target_os = "macos")]
+    {
+        copy_dir(
+            Path::new("../macos-redirector/MitmproxyAppleTunnel.app/"),
+            Path::new("/Applications/MitmproxyAppleTunnel.app/"),
```

Comment:
This is not copying into the wheel but to `/Applications`, isn't it?

Also it would be super awesome if we could copy only the compiled binaries and not the entire directory. 😃